### ISSUE TITLE
Panic on formatting SQL with multibyte chars and leading non-printing characters

### ISSUE THREAD

**Issue by abonander** [shssoichiro/sqlformat-rs#7]
https://github.com/launchbadge/sqlx/issues/831

It looks like `sqlformat` chokes on the first query here:

```rust
let query = "\nSELECT 'главная'"; // bad query
// let query = "SELECT 'главная'"; // good query
// let query = "\nSELECT 'main'"; // good query
```

Panic location: `sqlformat-0.1.4\src\tokenizer.rs:157:37`

**Comment by shssoichiro** [comment]
Thanks for the report! Fixed in 0.1.5.

**Comment by Tirka** [comment]
Thank you!
